model/Image.php

```diff
@@ -123,25 +123,13 @@ public function getCMSFields() {
 	}
 
 	/**
-	 * An image exists if it has a filename.
-	 * Does not do any filesystem checks.
```

Comment:
It's an interesting case where a function has been documented with no care as to whether it's really right.

It's just "this code does something".

```php

/**
 * This function multiples the two arguments
 */
function divide($numerator, $denominator) {
return $numerator * $denominator;
}
```

Now, if that doc block was just added later to state what the function does, as opposed to document it's intended purpose, then we can say it's a bug, otherwise we'd say it's stupid naming but acts as intended.

The API for "exists" has already been set and shouldn't really change arbitrarally because old code changed it's behaviour; just because someone documented it incorrectly doesn't mean it's expected behaviour